### PR TITLE
Correct user info for CouchDB image; remove unneded cert/proxy hooks

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes https://github.com/dmwm/WMCore/issues/11980

Summary of changes is:
* built and uploaded a new image for `cmsweb/couchdb` with tag `3.2.2-stable`
* synchronize the id used inside the CouchDB image with what is expected in CMSWEB VM node
* remove proxy/certificate hooks, given that we don't use them in central services (only if starting replication)
* update README with start/stoping CouchDB container and checking status

PS.: During the migration, a script named `check_couchdb.py` was used to cross check which backup files had to be copied where. That script is no longer needed, but here is a commit/pointer to it, for future reference: https://github.com/dmwm/CMSKubernetes/blob/ffb19657b8a768b5a03557bb97021006270fb28b/docker/couchdb/check_couchdb.py

NOTE: I forgot updating `vm.args` file to have a proper identification of the couchdb node. I am now discussing with the developers to see if we can easily make that change or not. Further information in this documentation: https://docs.couchdb.org/en/stable/setup/cluster.html#configure-and-test-the-communication-with-erlang

### CODE REVIEW

File: docker/couchdb/docker-run.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+--network=host \
+--hostname=$(hostname -f) \
+--name=couchdb \
+--mount type=bind,source=$COUCH_SECR_DIR,target=/etc/secrets \
+--mount type=bind,source=$COUCH_DB_DIR,target=$COUCH_DB_DIR \
+--mount type=bind,source=$COUCH_STAGING_DIR,target=$COUCH_STAGING_DIR \
+--mount type=bind,source=$COUCH_LOGS_DIR,target=$COUCH_LOGS_DIR \
+"
+
+couch_tag=3.2.2-v6
```
Comment (amaltaro) [inline docker/couchdb/docker-run.sh:32]: FIXME: update to a stable tag.

**Comment by amaltaro** [comment]
@arooshap Aroosha, I updated the description of this PR and will soon remove any scripts that are no longer relevant (those that were needed only for the migration). Once I clear things up, I will ask for your review.

I will also go ahead and create a stable image tag for this CouchDB, next time we stop the service, we can pull in the new image then.

**Comment by amaltaro** [comment]
@arooshap I built/uploaded a stable release, removed a no-longer-needed script and updated the README file. From my side, these changes are good to go in. Could you please review it? Thanks

**Review by arooshap — APPROVED** [review]
The changes look good to me, thanks Alan. Feel free to merge it.

**Pull request merged**